sdv/metadata/metadata.py

```diff
@@ -153,6 +153,20 @@ def detect_from_dataframe(
         metadata.detect_table_from_dataframe(table_name, data, infer_sdtypes, infer_keys)
         return metadata
 
+    def validate_data(self, data):
```

Comment:
why are we adding this?

Reply:
`MultiTableMetadata` requires `data` to be a [dictionary](https://github.com/sdv-dev/SDV/blob/391fd824fad082a97207705c522a61b7ae0ca68a/sdv/metadata/multi_table.py#L864-L865). Without this change single table synthesizers don't support taking data as a dataframe.

Reply:
Can we have the single table synthesizers handle this by passing in a dictionary instead?